stable/spinnaker/values.yaml

```diff
@@ -40,6 +40,10 @@ halyard:
   ## run by the init container before halyard is started.
   # additionalInitScript: |-
 
+  ## Uncomment if you want to add annotations on halyard and install-using-hal pods
+  additionalAnnotations:
```

Comment:
This needs to be commented out.

Reply:
good point, @viglesiasce ! made the change